pkg/cloudprovider/aws/suite_test.go

```diff
@@ -128,6 +134,7 @@ var _ = BeforeSuite(func() {
 })
 
 var _ = AfterSuite(func() {
+	stop()
```

Comment:
I needed to do this otherwise the watch I set up in UserDataProvider kept a connection open with the API Server and the API Server wouldn't shut down, leading to the tests to fail. Kept running into this - https://github.com/kubernetes-sigs/controller-runtime/issues/1571